### PR TITLE
[#108] - finalize inputs

### PR DESCRIPTION
**Pull request by msd96drexel**

### CODE REVIEW

**Comment by pinkdragon1000** [comment]
set a min to birth year to...idk 1919 (100 years ago) lmao..and maybe set a max as well

**Comment by pinkdragon1000** [comment]
If you want the step to work you have to add it as a props to the input component.  Right now it doesn't have that so it's not working as expected for lets say `Annual Salary After Taxes`

File: frontend/src/Components/ClientProfileContent.js [hunk 1]
```diff
@@ -132,41 +135,44 @@ class ClientProfileContent extends Component {
...
       return (
-        <div>
+        <React.Fragment>
           <h4>Income</h4>
           <div className='row'>
             <div className='column'>
               <InputComponent
                 type='number'
-                min='0'
+                min='0.00'
```
Comment (pinkdragon1000) [inline frontend/src/Components/ClientProfileContent.js]: The min is defaulting to 0 and not showing the decimal places for 0.00.  Also you should follow the {number} format like you used above.

File: frontend/src/Components/ClientProfileContent.js [hunk 2]
```diff
@@ -132,41 +135,44 @@ class ClientProfileContent extends Component {
...
                 onChange={props.handleChange}
+                step='0.01'
                 required
               />
             </div>
             <div className='column'>
               <InputComponent
                 type='number'
-                min='0'
+                min='0.00'
```
Comment (pinkdragon1000) [inline frontend/src/Components/ClientProfileContent.js]: Same thing here.  Refer to the previous comment.

File: frontend/src/Components/ClientProfileContent.js [hunk 3]
```diff
@@ -265,33 +276,47 @@ class ClientProfileContent extends Component {
...
+                label="Partner's Last Name"
                 name='spouselname'
-                placeholder="Type in your spouse's last name"
+                placeholder="Type in your partner's last name"
                 value={props.spouselname}
                 onChange={props.handleChange}
               />
+              <InputComponent
+                type='number'
+                min='0.00'
```
Comment (pinkdragon1000) [inline frontend/src/Components/ClientProfileContent.js]: Here too

File: frontend/src/Components/ClientProfileContent.js [hunk 4]
```diff
@@ -306,27 +331,30 @@ class ClientProfileContent extends Component {
...
-                value={props.childlname}
-                name='childlname'
+                type='number'
+                label="Child's Birth Year"
+                name='spousebday'
+                placeholder="Type in your child's birth year"
+                value={props.childbday}
                 onChange={props.handleChange}
+                min={1}
+                max={9999}
```
Comment (pinkdragon1000) [inline frontend/src/Components/ClientProfileContent.js:352]: - You shouldn't be able to enter in 1 for a birth year...and the max shouldn't be after this year.  I don't understand the logic of using those min and max values.  Maybe do the following: 
`max={new Date().getFullYear()-20} ` 
Let's say they must be at least 20 to use this application this ensures that you can't enter any year after 2000.  Similarly we can do maybe:
 `min={new Date().getFullYear()-100}`
Then you can't click down past 100 years ago
- Change the child's birth year input component `name=childbday`

File: frontend/src/Components/ClientProfileContent.js [hunk 5]
```diff
@@ -88,11 +88,14 @@ class ClientProfileContent extends Component {
...
-                type='date'
-                label='Date of Birth'
+                type='number'
+                label='Birth Year'
                 value={props.bday}
+                placeholder='Type in your birth year'
                 name='bday'
                 onChange={props.handleChange}
+                min={1}
+                max={9999}
```
Comment (pinkdragon1000) [inline frontend/src/Components/ClientProfileContent.js:98]: Refer to the comment about child bday.  The same thing applies here.

**Review by pinkdragon1000 — CHANGES_REQUESTED** [review]

File: frontend/src/Components/ClientProfileContent.js (hunk 4 above)
Comment (msd96drexel) [reply frontend/src/Components/ClientProfileContent.js:352]: for a better UX, we'll make this a drop-down

**Pull request merged**